packages/datadog_flutter_plugin/lib/src/datadog_sdk_method_channel.dart

```diff
@@ -41,18 +41,19 @@ class DatadogSdkMethodChannel extends DatadogSdkPlatform {
             logCallback(call.arguments as String);
             break;
         }
-        return Future.value();
+        return Future<void>.value();
       });
     }
 
-    await methodChannel.invokeMethod('initialize', {
+    await methodChannel.invokeMethod<void>('initialize', {
       'configuration': configuration.encode(),
       'setLogCallback': logCallback != null,
     });
   }
 
   @override
   Future<void> flushAndDeinitialize() {
-    return methodChannel.invokeMethod('flushAndDeinitialize', {});
+    return methodChannel
+        .invokeMethod('flushAndDeinitialize', <Object?, Object?>{});
```

Comment:
Let's be consistent with `<String, Object?>`.  Everything I've written in the native code expects a `Map<String, Object?>` even it it's empty.